src/Core/src/Platform/Android/LocalizedDigitsKeyListener.cs

```diff
@@ -19,13 +20,11 @@ public class LocalizedDigitsKeyListener : NumberKeyListener
 
 		static char GetDecimalSeparator()
 		{
-			if (!(NumberFormat.Instance is DecimalFormat format))
+			if (!(NumberFormat.Instance is DecimalFormat))
```

Comment:
This whole change is scary. It says in the android docs:

> To obtain a NumberFormat for a specific locale, including the default locale, call one of NumberFormat's factory methods, such as getInstance().

So is that wrong? The thing is not localized?

What is the value for the locales that are using a comma?

If this whole thing is not needed, then we may as well just delete this if block. Is NumberFormat.Instance ever null?

Reply:
I see the localized is just for Android 26+, so yeah, this instance thing is not useful - there is a locale overoad we could use on 26+

But then what about 21-25? Probably easier to just keep our code and skiap all of this number format stuff.